app/views/people/index.html.erb

```diff
@@ -33,5 +33,14 @@
         <td><%= edit_button(person.user, "", "fa fa-user-edit") if person.user_id %></td>
       </tr>
     <% end %>
+    <% if @pagy.pages > 1 %>
+      <tr>
+        <td></td>
+        <td></td>
+        <td colspan=2>
+          <%= pagy_nav(@pagy).html_safe %>
```

Comment:
I noticed that pagy ships with some [bulma specific helpers](https://ddnexus.github.io/pagy/extras/bulma). Not sure exactly what that provides, but am curious.

Reply:
I've not used `bulma` before so I'm happy to try out the `bulma` specific helpers. I'll see if one is obviously best.